ext/pg_result.c

```diff
@@ -404,6 +406,29 @@ pgresult_get(VALUE self)
 	return this->pgresult;
 }
 
+static VALUE pg_cstr_to_sym(char *cstr, unsigned int flags, int enc_idx)
+{
+	VALUE fname;
+	if( flags & PG_RESULT_FIELD_NAMES_SYMBOL ){
+#ifndef TRUFFLERUBY
```

Comment:
The code here looks correct for `:symbol` and `:string` on truffle ruby, but it looks like `:static_symbol` will produce strings.